querydsl-tooling/querydsl-ksp-codegen/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.github.openfeign.querydsl</groupId>
+    <artifactId>querydsl-tooling</artifactId>
+    <version>6.9-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>querydsl-ksp-codegen</artifactId>
+  <name>Querydsl - Kotlin KSP codegen support</name>
+
+  <properties>
+    <surefire.useManifestOnlyJar>false</surefire.useManifestOnlyJar>
```

Comment:
Why this is needed?

Reply:
`<surefire.useManifestOnlyJar>false</surefire.useManifestOnlyJar>`
You are referring to this line right?

Honestly i don't know what it's doing, i included it because kotlin-codegen had it.
I started out by copying their pom and modified the dependencies and some other stuff.
It's been a while since i used Maven so there may be parts that are not necessary.
I can take a closer look to see what can be removed.

Reply:
Ow, well, yeah, give it a shot, is makes no change, nuke it =)

Reply:
Only part of the pom i'm unsure about now is reference to dokka-maven-plugin.
I can see that the other Kotlin module is using it, it says online that it's related to API documentation.
I don't know how it's used or if it's used.